data/txts/tips/general_game.lua

```diff
@@ -76,7 +76,9 @@ tips = {
    {
       text = _("Zoom in/out with the mouse wheel or %1%/%2%. Press %3% to reset the zoom."):bformat(
```

Comment:
You need this to add the new shortcuts to the tips:

```
text = _("Zoom in/out with the mouse wheel or %1%/%2%. Press %3%/%4% for maximum/minimum zoom level. Press %5% to reset the zoom."):bformat(
wl.ui.get_shortcut("zoom_in"),
wl.ui.get_shortcut("zoom_out"),
wl.ui.get_shortcut("zoom_max"),
wl.ui.get_shortcut("zoom_min"),
wl.ui.get_shortcut("zoom_reset")),
seconds = 5
```

(increase `seconds` to give more time for reading)